Sends in disaster relief for the great firelock famine

chaoko99 opened the pull request:
That is, adds a lot of much needed firelocks to the torch.

Certainly my airlock needs a firelock, but surely this glass that *isn't even bulletproof* can hold a normal to null kPa differential or fire!

Fixes https://github.com/Baystation12/Baystation12/issues/17035 while I'm down there.

PsiOmegaDelta commented:
Are you sure you properly cleaned the maps?
Even with a mass-spam of airlocks it shouldn't result in thousands of changed lines.

chaoko99 commented:
I'll run the cleaner again.

xales commented on html/changelogs/Chaoko99-gibfirelocks.yml:
```diff
@@ -0,0 +1,38 @@
+#################################
+
+# Your name.  
+author: Unknown
```
Missed the author again.

chaoko99 replied:
,,, How does this keep happening?

chaoko99 replied:
Handled.

chaoko99 commented:
Ran the cleaner. Same result. Odd.

BlueNexus commented:
@chaoko99 Does this follow the torch's mapping guidelines/design considerations?
https://baystation12.net/forums/threads/map-development.1798/
A lot of the missing firelocks are intentional, on maintenance doors, etc.

Raptor1628 commented:
For the record, design considerations are only that, considerations. If things need to be changed, change them. I no longer have the authority to make a yea or nay, however seeing as the thicker hull of the torch keeps fire in a lot better than the Exodus hull, this seems like a reasonable change.

Chiquiii commented:
I know whats wrong with the map merger chaoko. When you start the map merge which files does it merge?

chaoko99 commented:
Out of home atm, I'll check back on that in a bit. @orelban

On Wed, Apr 26, 2017 at 9:53 AM, orelbon <notifications@github.com> wrote:

> I know whats wrong with the map merger chaoko. When you start the map
> merge which files does it merge?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Baystation12/Baystation12/pull/17030#issuecomment-297473848>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEfPBF1sLdQiyzDUJzvUYmyrnIXL316hks5rz3aVgaJpZM4NHJ0z>
> .
>

chaoko99 commented:
@orelbon https://pastebin.com/TjA4LTnd All of them? (HAngs on 10/11, so I deleted them.

Chiquiii commented:
That's weird, then it should be working right.

chaoko99 commented:
At any rate: We've merged map changes. I have to restart anyways.

On Thu, Apr 27, 2017 at 12:40 AM, orelbon <notifications@github.com> wrote:

> That's weird, then it should be working right.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Baystation12/Baystation12/pull/17030#issuecomment-297636896>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEfPBP-7cWDd8hjpJjpGhpINLTMdCnHcks5r0EZ8gaJpZM4NHJ0z>
> .
>

xales commented:
I believe the point of mapmerge is to prevent having to re-map things just because the file was changed...

chaoko99 commented:
Well, I've not a single clue what's wrong, at any rate.

On Thu, Apr 27, 2017 at 12:57 PM, Jacob Greenfield <notifications@github.com
> wrote:

> I believe the point of mapmerge is to prevent having to re-map things just
> because the file was changed...
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Baystation12/Baystation12/pull/17030#issuecomment-297822238>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEfPBAwCDTU7v1R6rTkbDq7GJr_cYXB4ks5r0PM2gaJpZM4NHJ0z>
> .
>

chaoko99 commented:
Found the problem. I'll need to start over, because I am a doof who didn't prepare the map before making my edits. I edited them first.

chaoko99 commented:
Conflicts resolved. @xales

xales commented:
Looks good. Probably should squash it, but also looking for a lore pass from @SiegDerMaus here to make sure it's good to go.

chaoko99 committed:
Changelog.


Sends in disaster relief for the great firelock famine.


Adds firelocks to deck 4. Fixes some firelocks from deck 2.

Also fixes the surgery holosign switch, as well as places a single double airlock across the chapel front door.

chaoko99 commented:
Squashed.

Hubblenaut commented:
This puts in air shutters on all maint hatches? Good, I was going to do that at some point as well. Had a long time coming, the missing air shutters serve nothing else but causing more work for Engineers by constant spreading of breaches/fires.

SiegDerMaus commented:
I guess I'll, tentatively, approve this. Since, as Rappy said, the missing firelocks were intentional. There's still nothing worse than opening a random door and having a face full of hard vacuum. If this makes the Torch too fire/breachproof, however, that may need to be fixed.

ghost commented:
The maps need updated so that they may be merged, @chaoko99

chaoko99 commented:
A A A A A

On Mon, May 1, 2017 at 5:10 PM, SkyMarshal <notifications@github.com> wrote:

> The maps need updated so that they may be merged, @chaoko99
> <https://github.com/chaoko99>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Baystation12/Baystation12/pull/17030#issuecomment-298465300>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEfPBDCHdYqW9JTgtMtGzHDDyworE6Noks5r1nSQgaJpZM4NHJ0z>
> .
>

chaoko99 committed:
Resolves Torch-4 conflict

chaoko99 commented:
Closing this so I may restart.
